### ISSUE TITLE
license agreement page coming again and again

### ISSUE THREAD

**Issue by ranjan-2209** [NuGet/Home#7407]
## Details about Problem

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe): VS UI

NuGet Package Manager   4.6.0

Microsoft Visual Studio Professional 2017 
Version 15.8.7
VisualStudio.15.Release/15.8.7+28010.2046
Microsoft .NET Framework
Version 4.7.03056

OS version (i.e. win10 v1607 (14393.321)): Windows 10 Enterprise 1803 17134.345

Worked before? If so, with which NuGet version: No

## Detailed repro steps so we can see the same problem

1. Whenever we install Nuget package, there should be a checkox telling not to ask for license agreement popup again

**Comment by nkolev92** [comment]
Would you want to disable the license agreement page for all packages? 

I'm not sure if something like this is feasible from a legal stand-point. 

//cc @karann-msft @anangaur would know better.

**Comment by ranjan-2209** [comment]
yes,

something like chrome also asks "Don't show this pop up again"

and we have to tick the checkbox.

**Comment by nkolev92** [comment]
@ranjan-ranjan 

Yes, I understand that, I just make sure I fully understood your request. 

I'll wait for the @karann-msft and @anangaur to address my concern before we triage this further.

**Comment by anangaur** [comment]
I am exploring getting rid of the pop up message altogether: https://github.com/NuGet/Home/issues/7439

**Comment by nkolev92** [comment]
@anangaur 
Should we close this issue then? Or leave it open?

**Comment by karann-msft** [comment]
@nkolev92 we should close it if we decide to implement #7439 else keep it open
